amulet/level/formats/anvil_world/format.py

```diff
@@ -391,6 +392,48 @@ def _get_raw_chunk_data(
         """
         return self._get_dimension(dimension).get_chunk_data(cx, cz)
 
+    def get_players(self) -> Generator[str, None, None]:
+        """
+        Returns a generator of all player ids that are present in the level
+        """
+        yield from (
+            os.path.splitext(pid)[0]
+            for pid in (
+                os.path.basename(f)
+                for f in glob.iglob(os.path.join(self.path, "playerdata", "*.dat"))
+            )
+        )
+
+    def get_player(self, player_id: str = LOCAL_PLAYER) -> Player:
+        """
+        Gets the :class:`Player` object that belongs to the specified player id
+
+        If no parameter is supplied, the data of the local player will be returned
+
+        :param player_id: The desired player id
+        :return: A Player instance
+        """
+        if player_id == LOCAL_PLAYER and "Player" in self.root_tag["Data"]:
+            return Player(
+                player_id,
+                tuple(map(lambda t: t.value, self.root_tag["Data"]["Player"]["Pos"])),
+                tuple(
+                    map(lambda t: t.value, self.root_tag["Data"]["Player"]["Rotation"])
+                ),
+            )
+        elif player_id == LOCAL_PLAYER:
+            raise KeyError("Local player doesn't exist")
+
+        path = os.path.join(self.path, "playerdata", f"{player_id}.dat")
+        if not os.path.exists(path):
+            raise KeyError(f"Player {player_id} doesn't exist")
+        player_nbt = nbt.load(path)
+        return Player(
+            player_id,
+            tuple(map(lambda t: t.value, player_nbt["Pos"])),
+            tuple(map(lambda t: t.value, player_nbt["Rotation"])),
+        )
```

Comment:
This would be better structured
```py
if player_id == LOCAL_PLAYER:
    if "Player" in self.root_tag["Data"]:
        # load local player
    else:
        raise KeyError("Local player doesn't exist")
else:
    # load other player